--- clamav.c	2007-09-25 17:50:25.000000000 +0200
+++ clamav.c.oden	2008-04-18 09:55:26.000000000 +0200
@@ -112,9 +112,6 @@ PHP_INI_BEGIN()
     STD_PHP_INI_ENTRY("clamav.maxfilesize", "0", PHP_INI_ALL, OnUpdateLong, 
 					  maxfilesize, zend_clamav_globals, clamav_globals)
 
-    STD_PHP_INI_ENTRY("clamav.maxratio", "200", PHP_INI_ALL, OnUpdateLong, 
-					  maxratio, zend_clamav_globals, clamav_globals)
-
 PHP_INI_END()
 /* }}} */
 
@@ -127,7 +124,6 @@ static void php_clamav_init_globals(zend
 	clamav_globals->maxfiles      = 0;
 	clamav_globals->archivememlim = 0;
 	clamav_globals->maxfilesize   = 0;
-	clamav_globals->maxratio      = 0;
 }
 /* }}} */
 
@@ -161,7 +157,6 @@ PHP_MINIT_FUNCTION(clamav)
 	limits.maxfiles      = CLAMAV_G(maxfiles); 		/* max files */	
 	limits.maxfilesize   = CLAMAV_G(maxfilesize);	/* maximal archived file size */
 	limits.maxreclevel   = CLAMAV_G(maxreclevel);	/* maximal recursion level */
-	limits.maxratio      = CLAMAV_G(maxratio);		/* maximal compression ratio */
 	limits.archivememlim = CLAMAV_G(archivememlim);	/* disable memory limit for bzip2 scanner */
 
 	/*  Register ClamAV scan options, they're also availible inside
@@ -406,12 +401,11 @@ PHP_FUNCTION(cl_scanfile)
 PHP_FUNCTION(cl_setlimits)
 {
 	/* number of arguments */
-	const int NUM_ARGS	= 5;
+	const int NUM_ARGS	= 4;
 	
 	int maxfiles;
 	int maxfilesize;
 	int maxreclevel;
-	int maxratio;
 	int archivememlim;
 
 	/* argument checking */
@@ -421,9 +415,9 @@ PHP_FUNCTION(cl_setlimits)
 	}
 
 	/* argument parsing */
-	if (zend_parse_parameters(NUM_ARGS TSRMLS_CC, "lllll", 
+	if (zend_parse_parameters(NUM_ARGS TSRMLS_CC, "llll", 
 							  &maxfiles, &maxfilesize,
-							  &maxreclevel, &maxratio, 
+							  &maxreclevel, 
 							  &archivememlim ) == FAILURE) 
 		return FAILURE;
 	
@@ -432,7 +426,6 @@ PHP_FUNCTION(cl_setlimits)
 	limits.maxfiles      = maxfiles; 		/* max files */	
 	limits.maxfilesize   = maxfilesize;		/* maximal archived file size */
 	limits.maxreclevel   = maxreclevel;		/* maximal recursion level */
-	limits.maxratio      = maxratio;		/* maximal compression ratio */
 	limits.archivememlim = archivememlim;	/* disable memory limit for bzip2 scanner */
 
 	RETURN_TRUE;	
